### ISSUE TITLE
lt can't work on this site dexfilter.com

### ISSUE THREAD

**Issue by zhajingwen** [FlareSolverr/FlareSolverr#712]
### Have you checked our README?

- [X] I have checked the README

### Is there already an issue for your problem?

- [X] I have checked older issues, open and closed

### Have you checked the discussions?

- [X] I have read the Discussions

### Environment

```markdown
- FlareSolverr version:
- Last working FlareSolverr version:
- Operating system:
- Are you using Docker: [yes/no]
- FlareSolverr User-Agent (see log traces or / endpoint):
- Are you using a VPN: [yes/no]
- Are you using a Proxy: [yes/no]
- Are you using Captcha Solver: [yes/no]
- If using captcha solver, which one:
- URL to test this issue:
```

### Description

when l want to request this website `https://dexfilter.com` by the way of container
It can not work

### Logged Error Messages

```text
curl -L -X POST 'http://localhost:8191/v1' \
-H 'Content-Type: application/json' \
-d '{
  "cmd": "request.get",
  "url":"https://dexfilter.com/",
  "maxTimeout": 60000
}'
```

```
{"status": "error", "message": "Error: Error solving the challenge. Timeout after 60.0 seconds.", "startTimestamp": 1677141832103, "endTimestamp": 1677141892875, "version": "3.0.2"}
```
```


### Screenshots

![Uploading image.png…]()

**Comment by ilike2burnthing** [comment]
Provide the environment information requested.

**Comment by zhajingwen** [comment]
> https://dexfilter.com

- FlareSolverr version: lasted
- Last working FlareSolverr version: lasted
- Operating system: CentOS Linux release 7.9.2009 (Core)
- Are you using Docker: [yes/no] yes
- FlareSolverr User-Agent (see log traces or / endpoint): default ua
- Are you using a VPN: [yes/no] no
- Are you using a Proxy: [yes/no] no
- Are you using Captcha Solver: [yes/no] no
- If using captcha solver, which one:
- URL to test this issue: https://dexfilter.com

**Comment by ilike2burnthing** [comment]
`lasted` (assuming you mean 'latest') is not a version. Log shows v3.0.2 so going by that.

Do the source of the request and FlareSolverr share the same public IP and use the same DNS?

Have you disabled IPv6?

Can you run FlareSolverr locally, rather than through Docker?

**Comment by GravuTrad** [comment]
https://github.com/FlareSolverr/FlareSolverr/issues/667

**Comment by zhajingwen** [comment]
> `lasted` (assuming you mean 'latest') is not a version. Log shows v3.0.2 so going by that.
> 
> Do the source of the request and FlareSolverr share the same public IP and use the same DNS?
> 
> Have you disabled IPv6?
> 
> Can you run FlareSolverr locally, rather than through Docker?

yes, is 'latest' 
`Do the source of the request and FlareSolverr share the same public IP and use the same DNS?` yes is same
`Have you disabled IPv6?` no, why do that?
`Can you run FlareSolverr locally, rather than through Docker?` no, l have try run through Docker directly

l think my ip of Cloud Host and all proxies are all locked through my research in nearly 2 days, l want to buy some new proxy ips to have a try

**Comment by ilike2burnthing** [comment]
Disable IPv6, otherwise you can end up with a case where the original request is made from an IPv4 IP and the FlareSolverr uses IPv6, and the cookie is invalid.

Failing that, as a test try running FlareSolverr on a local machine (e.g. your home computer) rather than your current VPS or seedbox.

**Comment by zhajingwen** [comment]
OK thank you very much , l will have a try

**Comment by rix1337** [comment]
Looks like the issue fixed in #715

**Comment by ilike2burnthing** [comment]
v3.0.3 should resolve the issue. Retry a few times if it continues after updating.
